multiqc/modules/multivcfanalyzer/multivcfanalyzer.py

```diff
@@ -0,0 +1,239 @@
+        plot = self.addTable())
+        
+        # Add MultiVCFAnalyzer Barplot Section
+        self.add_section (
+        name = 'Call statistics barplot',
+        anchor = 'mvcf-barplot',
+        plot = self.addBarplot()
+        )
+
+
+    def parse_data(self, f):
+        try:
+            data = json.load(f['f'])
+        except Exception as e:
+            log.debug(e)
+            log.warn("Could not parse MultiVCFAnalyzer JSON: '{}'".format(f['fn']))
+            return
+
+        # Parse JSON data to a dict
+        for s_name, metrics in data.get('metrics', {}).items():
+            s_clean = self.clean_s_name(s_name, f['root'])
+            if s_clean in self.mvcf_data:
+                log.debug("Duplicate sample name found! Overwriting: {}".format(s_clean))
+
+            self.add_data_source(f, s_clean)
+            self.mvcf_data[s_clean] = dict()           
+            for snp_prop, value in metrics.items():
+                self.mvcf_data[s_clean][snp_prop] = value
+
+    #Compute % heterozygous snp alleles and add to data
+    def compute_perc_hets(self):
+        """Take the parsed stats from MultiVCFAnalyzer and add one column per sample """
+        for sample in self.mvcf_data:
+            try:
+                self.mvcf_data[sample]['Heterozygous SNP alleles (percent)'] = (self.mvcf_data[sample]['SNP Calls (het)'] / self.mvcf_data[sample]['SNP Calls (all)'])*100
+            except ZeroDivisionError:
+                self.mvcf_data[sample]['Heterozygous SNP alleles (percent)'] = 'NA'
+
+    def addSummaryMetrics(self):
+        """ Take the parsed stats from MultiVCFAnalyzer and add it to the main plot """
+
+        headers = OrderedDict()
+        headers['SNP Calls (all)'] = {
+            'title': 'SNPs',
+            'description': 'Total number of non-reference calls made',
+            'scale': 'OrRd',
+            'shared_key': 'snp_call'
+        }
+        headers['SNP Calls (het)'] = {
+            'title': 'Het SNPs',
+            'description': 'Total number of non-reference calls not passing homozygosity thresholds',
+            'scale': 'OrRd',
+            'hidden': True,
+            'shared_key': 'snp_call'
+        }
+        headers['Heterozygous SNP alleles (percent)'] = {
+            'title': '% Hets',
+            'description': 'Percentage of heterozygous SNP alleles',
+            'scale': 'OrRd',
+            'shared_key': 'snp_call'
+        }
+        self.general_stats_addcols(self.mvcf_data, headers)
+
+    def addTable(self):
+        """ Take the parsed stats from MultiVCFAnalyzer and add it to the MVCF Table"""
+        headers = OrderedDict()
+
+        headers['allPos'] = {
+            'title': 'Bases in SNP Alignment',
+            'description': 'Length of FASTA file in base pairs (bp)',
+            'scale': 'BuPu',
+            'shared_key': 'calls'
```

Comment:
These will always be whole numbers right? So this for all columns which are integers:
```suggestion
            'shared_key': 'calls',
            'format': '{:,.0f}'
```